### ISSUE TITLE
Printing of settings with a list value misses a delimiter

### ISSUE THREAD

**Issue by andy71** [sibalzer/impfbot#119]
When printing the settings, every list value is just a concatenated string of the list entries, without any delimiter.

Please replace this part of the code within settings.py with a simple join...

```
                if isinstance(value, list):
                    value = ', '.join(value)
#                    list_str = ""
#                    for entry in value:
#                        list_str += str(entry)
#                    value = list_str
```
The line behind the "isinstance" is the new one. Old lines which should be deleted are commented out.

**Comment by andy71** [comment]
I added a pull request #120 to this.

Am I correct in assuming that a pull request or an issue is sufficient, but not both?

**Comment by paulypeter** [comment]
> Am I correct in assuming that a pull request or an issue is sufficient, but not both?

Writing both is fine! You followed the usual procedure, which is:

1. If you notice a bug or have a feature request, you open an issue detailing it. If you want to fix it yourself, you can say so in the issue. If not, that's fine as well!
2. If you do want to fix it, you fork the repo and implement your solution there.
3. You create a pull request from your repo to the upstream repo.

For a better overview, it is not recommended to open a pull request without having a corresponding issue.

**Comment by sibalzer** [comment]
closed with #120, thanks
